staff/marc-uson/precourse/Pasapalabra-2/help.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="styles/style.css">
+        <title>HELP: Pasapalabra</title>
+    
+        <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+    
+</head>
+<body>
+    <div class="container" id="helpWindow">
+
+            <h1 class="title" id="helpTitle">Instrucciones del juego:</h1>
+            <ol id="instructions">
+                <li class="help">
+                    Bienvenido al juego de Pasapalabra! Para empezar a jugar pulsar el boton "jugar!".
+                    Debes de contestar bien a las preguntas para completar el rosco y ganar puntos antes 
+                    del tiempo indicado.
+                </li>
+                <li class="help">
+                    Pulsa intro para enviar la palabra que escribas. si es correcta, la letra 
+                    correspondiente cambiará a verde y ganarás un punto, si es incorrecta cambiará 
+                    a rojo sin sumar puntos. si quieres dejar la palabra para más tarde, pulsa la tecla 
+                    de espacio o escribe pasapalabra y esa pregunta se guardará para el final.</li>
+                <li class="help">
+                    Cuando termines con el rosco o se termine el tiempo entrarás en el ranking. 
+                    Tambien puedes terminar la partida pulsando la tecla escape o pulsando el boton "finalizar juego" 
+                    pero se te restarán 5 puntos. 
+                </li>
+            </ol>
+
+            <h2 style="text-align: center">SUERTE!</h3>
```

Comment:
you could add this class into `styles.css`

Reply:
si, pero al ser una pagina html estatica donde sale un texto que no varia, lo añadi aqui para no engordar mi "styles.css"